File: day7/day7es1.js
class Studente {
    constructor(nome, matricola) {
        this.nome = nome;
        this.matricola = matricola;
        this.voti = [];
    }
}


const studente1 = new Studente("Mario Rossi", 12345);
console.log(`L'oggetto studente1 è stato creato con nome: ${studente1.nome}`);

File: day7/day7es1.py
class Studente:
    def __init__(self, nome, matricola):
        self.nome = nome
        self.matricola = matricola
        self.voti = []


studente1 = Studente("Mario Rossi", 12345)
print(f"L'oggetto studente1 è stato creato con nome: {studente1.nome}")

